modules/image-loading-optimization/module.php

```diff
@@ -0,0 +1,346 @@
+<?php
+namespace Elementor\Modules\ImageLoadingOptimization;
+
+use Elementor\Core\Base\Module as BaseModule;
+use Elementor\Core\Experiments\Manager as Experiments_Manager;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+class Module extends BaseModule {
+	/**
+	 * @var string The experiment name.
+	 */
+	const EXPERIMENT_NAME = 'e_image_loading_optimization';
+
+	/**
+	 * @var int Minimum square-pixels threshold.
+	 */
+	private $min_priority_img_pixels = 50000;
+
+	/**
+	 * @var int The number of content media elements to not lazy-load.
+	 */
+	private $omit_threshold = 3;
+
+	/**
+	 * @var array Keep a track of images for which loading optimization strategy were computed.
+	 */
+	private static $image_visited = [];
+
+	/**
+	 * Get Module name.
+	 */
+	public function get_name() {
+		return 'image-loading-optimization';
+	}
+
+	/**
+	 * Get experimental data.
+	 *
+	 * @return array Experimental settings.
+	 */
+	public static function get_experimental_data() {
+		return [
+			'name' => static::EXPERIMENT_NAME,
+			'title' => esc_html__( 'Optimize Image Loading', 'elementor' ),
+			'tag' => esc_html__( 'Performance', 'elementor' ),
+			'description' => esc_html__( 'Applying `fetchpriority="high"` on LCP image and lazy loading on images below the fold.', 'elementor' ),
```

Comment:
Could this description be more clear? I find it a bit cryptic now.

Reply:
Applying `fetchpriority="high"` on LCP image and lazy loading on images below the fold to improve performance scores.

@kt-12

Reply:
This should be "Beta" and set to new sites from v3.17 (min version)

See references: https://github.com/elementor/elementor/pull/23876/files#diff-719a36795ac3daf2c97e2d1a89200ede89ae178f95ad1b85cb0187bd47d4409dR150

@kt-12

Reply:
Thanks for these suggestions, will do.